src/entities/Place/utils.ts

```diff
@@ -4,6 +4,8 @@ import { v4 as uuid } from "uuid"
 
 import { PlaceAttributes } from "./types"
 
+const DECENTRALAND_URL = "https://play.decentraland.org"
```

Comment:
whenever we have a target URL, we should have the option to change it using an environment variable. In this case:

```suggestion
const DECENTRALAND_URL =  process.env.GATSBY_DECENTRALAND_URL || process.env.DECENTRALAND_URL || "https://play.decentraland.org"
```